WordPress/Jetpack/Resources/release_notes.txt

```diff
@@ -1,15 +1 @@
-With 17.6, you can add the Audio Block to any WordPress.com site, whatever plan you’re on – including the Free Plan. The Audio Block makes it easy to add podcasts, music, and sound files to your posts and pages.
-
-We’ve been hard at work wrangling UX improvements, too:
-
-When you assign a parent page to a page without a title, you’ll now see a ‘(no title)’ placeholder instead of a blank line, giving you a gentle nudge to name the page.
-
-You’ve never been able to edit Post Archive pages because they’re automatically generated. In 17.6, you won’t be able to open them in the editor, either, saving you a wasted trip.
-
-We’ve also been working on fixing things:
-
-We took care of some notices that had been overlapping the ActionSheet, including one that displays Site Icon controls, another displaying the QuickStart Removal, and a third that displays More Actions in the editor.
-
-Last up, we squashed a bug that sometimes caused the app to crash during comment moderation and another to fix a login error for WordPress.org sites to show inline.
-
-That’s all for 17.6. See you in 17.7!
+The Block editor has a shiny new block Layout grid, and while we were digging around in the editor code, we also tweaked the code for our tablet-using friends. The pesky inserter button tablet view issue in the Block editor has now been fixed.
```

Comment:
Yes, it is.

Blogging Reminders is also present but we had an issue with it so I removed it from the copy, just in case. The issue is fixed now, but it will be out of the release notes (it's not merged yet anyway).